drivers/ethernet/eth_enc424j600.c

```diff
@@ -484,7 +485,6 @@ static void enc424j600_rx_thread(struct enc424j600_runtime *context)
 			}
 		} else {
 			LOG_ERR("Unknown Interrupt, EIR: 0x%04x", eir);
-			continue;
```

Comment:
NACK, this is intended as explained [here]( https://github.com/zephyrproject-rtos/zephyr/issues/35091#issuecomment-839813047)

Reply:
This should be behind a Kconfig option that is disabled by default. A driver should not screw itself up because it received an event that it can safely ignore. Query: should an AT modem completely shut down if it receives a wrong command or emit an error and continue working as normal? This is useful for driver development and debugging but is nothing but extremely unhelpful in a production-grade firmware for an IoT device

Reply:
No, it must not be ignored. To be clear, alternative to continue is k_oops()

Reply:
This is not in-line with any other ethernet driver in zephyr nor the encx24j600 driver in the linux kernel, it is staying to keep all these drivers operating in the same expected way.

Reply:
It is nonsense. Do not change it, do not introduce new config option. And please do not resolve unresolved conversation.